src/components/Review/ReviewSubmit.tsx

```diff
@@ -0,0 +1,206 @@
+import React, { useState } from 'react'
+import { Button, Checkbox, Form, ModalProps, TextArea } from 'semantic-ui-react'
+import { ModalWarning } from '..'
+import strings from '../../utils/constants'
+import { Decision, ReviewStatus } from '../../utils/generated/graphql'
+import { useRouter } from '../../utils/hooks/useRouter'
+
+import messages from '../../utils/messages'
+import { AssignmentDetailsNEW, FullStructure } from '../../utils/types'
+import useGetDecisionOptions, { DecisionOption } from './hooks/useGetDecisionOptions'
+import useSubmitReview from './hooks/useSubmitReview'
+import useUpdateReviewDecisionComment from './hooks/useUpdateReviewDecisionComment'
+
+type ReviewSubmitProps = {
+  structure: FullStructure
+  reviewAssignment: AssignmentDetailsNEW
+  scrollTo: (code: string) => void
+}
+
+const ReviewSubmit: React.FC<ReviewSubmitProps> = (props) => {
+  const { structure } = props
+  const thisReview = structure?.thisReview
+  const reviewDecision = thisReview?.reviewDecision
+  const {
+    decisionOptions,
+    getDecision,
+    setDecision,
+    getAndSetDecisionError,
```

Comment:
I thought to name it in such a way where it's obvious that this will not only get the decision error state, but set it (it does a check if it's ok to submit in terms of decision state, and if it's not ok it sets the state to show as error in the ReviewDecision component)